spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -1659,5 +1659,45 @@ def test_wrong_std_module(ipyconsole, qtbot):
     os.remove(wrong_random_mod)
 
 
+@flaky(max_runs=3)
```

Comment:
Same question

Reply:
See my explanation above. The same one applies here.